gitbook/geostyler/stepSix.md

```diff
@@ -7,9 +7,9 @@ von der GeoStyler Bibliothek importiert und in der Variable `olParser` referenzi
 import OpenLayersParser from "geostyler-openlayers-parser";
 const olParser = new OpenLayersParser();
 
-OlParser
+olParser
   .readStyle(defaultOlStyle)
-  .then(gsStyle => console.log(gsStyle))
+  .then(gsStyle => console.log(gsStyle.output))
   .catch(error => console.log(error));
```

Comment:
The Promise returned by geostyler parsers does not throw errors anymore, so the `.catch()` part is obsolete